packages/gatsby-source-lever/src/normalize.js

```diff
@@ -142,7 +146,7 @@ exports.createNodesFromEntities = ({ entities, createNode }) => {
       children: [],
       internal: {
         type: `lever`,
-        contentDigest: digest(JSON.stringify(entity)),
+        contentDigest: createContentDigest(entity),
```

Comment:
you forget to remove this?:
https://github.com/gatsbyjs/gatsby/blob/807560593b24c1a412e0f9c2ffc79d70d150af21/packages/gatsby-source-lever/src/normalize.js#L14-L18
and change this:
https://github.com/gatsbyjs/gatsby/blob/807560593b24c1a412e0f9c2ffc79d70d150af21/packages/gatsby-source-lever/src/normalize.js#L39

Reply:
I did it on purpose the `createGraphQLNode` function is exported but unused in gatsby codebase. No documentation refers to it and I need to add a parameter to the function to make it work, this is a breaking change. I don't know how to do about it, any advice ?

Reply:
I just saw an [example](https://github.com/gatsbyjs/gatsby/pull/8992/files#diff-1a077ef3f1d488e1041a8fb4003d4f53R8) that imports directly `createContentDigest` that would be a good solution for not making a breaking change. What is the difference between the imported version and the injected version @DSchau ?

Reply:
@LekoArts suggested me to make the change anyway 😄

Reply:
also if it is not used: i would change it anyway to complete the Job "Use new createContentDigest" 

and later look if we remove `createGraphQLNode` if is not used anymore

Reply:
but wait what the @gatsbyjs/core team suggest as the best way

Reply:
note: 

my researches about `createGraphQLNode`

- [search all Github](https://github.com/search?q=createGraphQLNode&type=Code): there are some hits in code, but they always are defined and used in the same file
- [search Google](https://www.google.com/search?q="createGraphQLNode"): only 2 hits - one to github and one is just a SEO spam

Reply:
We should create the breaking change :) It would be weird if people are using it outside of gatsby.